pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
+| **Step 5**  | ❎           | **Datastore  clusters**                  | **Delete all clustering rules.**                           | - Clustering rules will have to be deleted before migration because this notion do not exist on the VCD side.                                                                                                          |                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
+| **Step 6**  | ❎           | **Memory over-commitment**               | **Scale up VCD.**                                          | - Foresee more resources on the VCD side or resize/optimize the VMs before migration on the vSphere on OVHcloud side. <br/> Because you won't be able to over-commit resources.                                        | [Modifying virtual machine resources](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/modify_hardware_configuration_of_vm)                                   |                                                                                                                                                
+| **Step 7**  | ❎           | **Resource pools** (sharing)             |                                                            | - Resource pools will be lost after migration because this notion no longer exists on the VCD side.                                                                                                                    |                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
```

Comment:
To add: "Consider using vApp instead after the migration." https://docs.vmware.com/en/VMware-Cloud-Director/10.6/VMware-Cloud-Director-Tenant-Guide/GUID-AC48FB5E-4ADC-4835-AACE-B949B297A147.html